mbuild/formats/lammpsdata.py

```diff
@@ -179,35 +202,41 @@ def write_lammpsdata(structure, filename, atom_style='full'):
             # Pair coefficients
             epsilons = [atom.epsilon for atom in structure.atoms]
             sigmas = [atom.sigma for atom in structure.atoms]
+            forcefields = [atom.type for atom in structure.atoms]
             epsilon_dict = dict([(unique_types.index(atom_type)+1,epsilon) for atom_type,epsilon in zip(types,epsilons)])
             sigma_dict = dict([(unique_types.index(atom_type)+1,sigma) for atom_type,sigma in zip(types,sigmas)])
-            data.write('\nPair Coeffs # lj\n\n')
+            forcefield_dict = dict([(unique_types.index(atom_type)+1,forcefield) for atom_type,forcefield in zip(types,forcefields)])
+            data.write('\nPair Coeffs # lj \n\n')
+            data.write('#\tepsilon\t\tsigma\n#\tkcal/mol\tAngstrom\n')
             for idx,epsilon in epsilon_dict.items():
-                data.write('{}\t{:.5f}\t{:.5f}\n'.format(idx,epsilon,sigma_dict[idx]))
+                data.write('{}\t{:.5f}\t\t{:.5f}\t\t# {}\n'.format(idx,epsilon,sigma_dict[idx],forcefield_dict[idx]))
 
             # Bond coefficients
             if bonds:
                 data.write('\nBond Coeffs # harmonic\n\n')
+                data.write('#\tk\t\treq\n# kcal/mol/Angstrom^2\tAngstroms\n')
                 for params,idx in unique_bond_types.items():
-                    data.write('{}\t{}\t{}\n'.format(idx,*params))
+                    data.write('{}\t{}\t\t{}\t\t# {}\n'.format(idx,*params))
 
             # Angle coefficients
             if angles:
                 data.write('\nAngle Coeffs # harmonic\n\n')
+                data.write('#\tk\t\ttheteq\n# kcal/mol/radians^2\tdegrees\n')
                 for params,idx in unique_angle_types.items():
-                    data.write('{}\t{}\t{:.5f}\n'.format(idx,*params))
+                    data.write('{}\t{}\t\t{:.5f}\t# {}\t{}\t{}\n'.format(idx,params[0],params[1],params[3][0],params[2],params[3][1]))
 
             # Dihedral coefficients
             if dihedrals:
                 data.write('\nDihedral Coeffs # opls\n\n')
+                data.write('#\tf1\t\tf2\t\tf3\t\tf4\n#\tkcal/mol\tkcal/mol\tkcal/mol\tkcal/mol\n')
                 for params,idx in unique_dihedral_types.items():
                     opls_coeffs = RB_to_OPLS(params[0],
                                              params[1],
                                              params[2],
                                              params[3],
                                              params[4],
                                              params[5])
-                    data.write('{}\t{:.5f}\t{:.5f}\t{:.5f}\t{:.5f}\n'.format(idx,*opls_coeffs))
+                    data.write('{}\t{:.5f}\t{:.5f}\t\t{:.5f}\t\t{:.5f}\t# {}\t{}\t{}\t{}\n'.format(idx,opls_coeffs[0],opls_coeffs[1],opls_coeffs[2],opls_coeffs[3],params[8],params[9],params[10],params[11]))
```

Comment:
Can we refactor these to try and stick with ~80-100 columns?
```suggestion
                    data.write('{}\t{:.5f}\t{:.5f}\t\t{:.5f}\t\t{:.5f}\t# {}\t{}\t{}\t{}\n'.format(idx,opls_coeffs[0],
                    																				opls_coeffs[1],	
    	                																			opls_coeffs[2], 	
                   																					opls_coeffs[3],
                    																				params[8],
                    																				params[9],
                    																				params[10],
                    																				params[11])
                    																	)
```
Something like this, but dont accept this modification, it was done using the markdown in github.